packages/node/test/e2e/snapshots/cli-general.spec.js.md

```diff
@@ -0,0 +1,48 @@
+# Snapshot report for `test/e2e/cli-general.spec.js`
+
+The actual snapshot is saved in `cli-general.spec.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## "--help" prints help
+
+> output of command "node [97m../../../src/cli.js --help[39m" does not match expected snapshot
+
+    {
+      code: undefined,
```

Comment:
does this refer to the exit code? (in all 3 snapshots)

Reply:
yes. 

unrelated, on L9m, I need to strip ANSI escapes before handing the string to be used in the assertion...